bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -22,6 +26,10 @@
 		<label>Siren</label>
 		<description>Status of the siren</description>
 		<category>Soundvolume</category>
+		<tags>
+			<tag>Status</tag>
+			<tag>SoundVolume</tag>
+		</tags>
```

Comment:
I have the strange feeling that "Alarm" would be more appropriate.

Reply:
I would opt to leave it as is. It is not an alarm itself e.g. a sound volume treshold that is exceeded. It is a status that the siren is making sound or not for whatever for alarm there is. (one of the eother channels)

Reply:
Ok